src/dlmalloc.rs

```diff
@@ -1783,8 +1790,9 @@ mod tests {
     #[test]
     // Test allocating, with a non-empty treemap, a specific size that used to
     // trigger an integer overflow bug
+    #[cfg(any(any(target_os = "linux", target_arch = "wasm32"), target_os = "macos"))]
     fn treemap_alloc_overflow_minimal() {
-        let mut a = DLMALLOC_INIT;
+        let mut a: Dlmalloc<Platform> = Dlmalloc::init();
```

Comment:
In this file: shouldn't need to specify type when calling `Dlmalloc::new()`

Reply:
I think here it does because it refers to the inner type, not the outer